app/services/users/manage_credits.rb

```diff
@@ -0,0 +1,50 @@
+module Users
+  class ManageCredits
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def initialize(user, user_params)
+      @user = user
+      @organization_id = user_params[:organization_id]
+      @add_credits_param = user_params[:add_credits]
+      @remove_credits_param = user_params[:remove_credits]
+      @add_org_credits_param = user_params[:add_org_credits]
+      @remove_org_credits_param = user_params[:remove_org_credits]
+    end
+
+    def call
+      add_credits if add_credits_param
+      remove_credits if remove_credits_param
```

Comment:
Since you handle credits for the `user` and credits for `orgs`.
Lets make this more explicitly like this:

```ruby
def call
  manage_user_credits
  manage_org_credits
end
```
And inside of each one of this new methods, you call the correct ones.